lib/plugins/backup.rb

```diff
@@ -3,8 +3,8 @@
 require_relative '../moonshot/creds_helper'
 
 module Moonshot::Plugins # rubocop:disable Documentation
-  # Monkeypatching TarWriter for our
-  # needs to improve clarity
+  # Defining new method for the ease of use in this
```

Comment:
_what does the original method do_ --> there is no original method, I can further elaborate why I thought this is useful tho.